iceoryx_posh/test/moduletests/test_roudi_portmanager.cpp

```diff
@@ -562,3 +562,22 @@ TEST_F(PortManager_test, PortDestroy)
         }
     }
 }
+
+TEST_F(PortManager_test, IntrospectionMemoryManagerNotInitialzed)
```

Comment:
This testcase has only tests on the `IceOryxRouDiMemoryManager`. The `PortManager_test` is mainly for having tests on the `PortManager` class.
My advise would be to create a new testfile which has only the `IceOryxRouDiMemoryManager`. You can adapt the testfile from an existing one. 
A good example how a testfile should look like can be found [here](https://github.com/eclipse-iceoryx/iceoryx/blob/master/doc/aspice_swe3_4/example/iceoryx_component/test/moduletests/test_example_base_class.cpp).
In the [documentation](https://github.com/eclipse-iceoryx/iceoryx/blob/master/doc/website/advanced/installation-guide-for-contributors.md#build-and-run-tests) is also described how you can add new testfiles to iceoryx.

Reply:
Done :)